components/LocationComparisonSection/LocationComparisonBanner.tsx

```diff
@@ -40,24 +44,36 @@ const LocationComparisonBanner: FunctionComponent<ComparisonWrapperProps> = ({
   };
 
   return (
-    <SpotlightBanner>
-      <SpotlightBannerAside>
-        <AddLocation active={!active} label={'Add Location'} onWidgetClick={onWidgetClick} />
-        <SpotlightMenuWithData
-          onWidgetClick={onWidgetClick}
-          countryName={countryName}
-          countryCode={countryCode}
-          spotlightMenu={active}
-        />
-        <SelectWithData show={active} countryCode={countryCode} onWidgetClick={onWidgetClick} />
-      </SpotlightBannerAside>
-      <SpotlightBannerMain>
+    <>
+      <SpotlightBanner>
+        <SpotlightBannerAside>
+          <AddLocation active={!active} label={'Add Location'} onWidgetClick={onWidgetClick} />
+          <SpotlightMenuWithData
+            onWidgetClick={onWidgetClick}
+            countryName={countryName}
+            countryCode={countryCode}
+            spotlightMenu={active}
+          />
+        </SpotlightBannerAside>
+        <SpotlightBannerMain>
+          <SelectWithData show={active} countryCode={countryCode} onWidgetClick={onWidgetClick} width="300px" />
+          <button type="button" className="countries__searched-cancel">
+            <style jsx>{`
+              .countries__searched-cancel {
+                display: ${active ? 'inline-block' : 'none'};
+              }
+            `}</style>
+            {<span>Cancel</span>}
+          </button>
+        </SpotlightBannerMain>
+      </SpotlightBanner>
+      <SpotlightBanner>
         <Tags onCloseTag={onCloseTag} updatedTags={locations} />
         <Button className={'button--compare'} onButtonClick={onClickCompare}>
```

Comment:
Remove from {}... if a string isn't going to require any extra work, u don't need to wrap it in curly braces.

Reply:
`OnButtonClick` seems redundant. Since we know it's a button, simple `onClick` is fine.

Reply:
Updated the string and `onClick` property name.